_layouts/default.html

```diff
@@ -3,19 +3,8 @@
 <!doctype html>
 <html>
 <head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="chrome=1">
+  {% include head.html %}
```

Comment:
This will make it easier to create separate layouts for the site, as liquid doesn't seem to support template inheritance out-of-the-box.